p2p/node/pubsubManager/utils.go

```diff
@@ -94,41 +94,52 @@ func NewTopic(genesis common.Hash, location common.Location, data interface{}) (
 	return t, nil
 }
 
-func TopicFromString(genesis common.Hash, topic string) (*Topic, error) {
+func TopicFromString(topic string) (*Topic, error) {
 	topicParts := strings.Split(topic, "/")
 	if len(topicParts) < 3 {
-		return nil, ErrMalformedTopic
+		return nil, errors.New("topic string should have 3 parts")
```

Comment:
Why create a new error definition? In any case the ErrMalformedTopic should either be updated or removed if not used here.

Reply:
done